### ISSUE TITLE
OnFinishUploaderHandler fire twice

### ISSUE THREAD

**Issue by GoogleCodeExporter** [bdshadow/gwtupload#88]
```
What steps will reproduce the problem?
1.Using SingleUploader
2.add an OnFinishUploadHandler
3.set AutoSubmit to true

What is the expected output? What do you see instead?
Trigger OnFinishUploadHandler for one time only when upload done.
But it trigger twice.

What version of the product are you using? On what operating system?
0.6.3

Please provide any additional information below.


```

Original issue reported on code.google.com by `fonghuan...@gmail.com` on 11 Jan 2011 at 6:13

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
Original comment by `manuel.carrasco.m` on 7 Feb 2011 at 6:29
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
Original comment by `manuel.carrasco.m` on 7 Feb 2011 at 6:30

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision r863.
```

Original comment by `manuel.carrasco.m` on 13 Feb 2011 at 12:30
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
thanks for the fixed.
```

Original comment by `fonghuan...@gmail.com` on 14 Feb 2011 at 1:42

**Comment by GoogleCodeExporter** [comment]
```
Hi manuel,
I am still facing this issue with the gwtupload-0.6.3-compat.jar in the 
downloads section
```

Original comment by `veeravel...@gmail.com` on 25 Feb 2011 at 12:45

**Comment by GoogleCodeExporter** [comment]
```
To add to my comment above the upload handler is fired twice not all the time 
but for few uploads only
```

Original comment by `veeravel...@gmail.com` on 25 Feb 2011 at 12:50

**Comment by GoogleCodeExporter** [comment]
```
It is fixed in gwtupload-0.6.4-SNAPSHOT.jar
```

Original comment by `manuel.carrasco.m` on 25 Feb 2011 at 1:18

**Comment by GoogleCodeExporter** [comment]
```
Got it..thanks for the prompt reply manuel
```

Original comment by `veeravel...@gmail.com` on 28 Feb 2011 at 5:59
